pkg/asset/ignition/machine/worker.go

```diff
@@ -59,3 +63,18 @@ func (a *Worker) Files() []*asset.File {
 	}
 	return []*asset.File{}
 }
+
+// Load returns the worker ignitions from disk.
+func (a *Worker) Load(f asset.FileFetcher) (found bool, err error) {
+	a.File = f.FetchByName(workerIgnFilename)
+	if a.File == nil {
+		return false, nil
+	}
+
+	var config igntypes.Config
```

Comment:
Why do we need to unmarshal to a temporary variable instead of unmarshaling directly to `a.Config`?